### PR TITLE
git-big-picture: new formula v0.9.0

### PR DESCRIPTION
**Pull request by vicamo**
Add a new formula for git-big-picture from https://github.com/esc/git-big-picture .

### CODE REVIEW

File: Library/Formula/git-big-picture.rb [hunk 1]
```diff
@@ -0,0 +1,18 @@
+require 'formula'
+
+class GitBigPicture < Formula
+  homepage 'https://github.com/esc/git-big-picture'
+  url 'https://github.com/esc/git-big-picture/archive/v0.9.0.tar.gz'
+  sha1 'd0265f32bcb9e6c1b7991e470e621c1931ee6790'
+
+  depends_on 'graphviz' => :recommended
+  depends_on 'git'
```
Comment (mikemcquaid) [inline Library/Formula/git-big-picture.rb]: If people are using Homebrew they already have Git installed to we don't need this.

File: Library/Formula/git-big-picture.rb [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+  def install
+    bin.install 'git-big-picture'
+  end
+
+  test do
+    system "#{bin}/git-big-picture", "-h"
```
Comment (mikemcquaid) [inline Library/Formula/git-big-picture.rb]: Can this test be modified to do something more substantial than e.g. `--version` or `--help`? See `cmake.rb` for an example of an application formula with a good test and `tinyxml2.rb` for an example of a library formula with a good test. Thanks!

Comment (mikemcquaid) [inline Library/Formula/git-big-picture.rb]: Perhaps create a Git repository in the current (temporary) directory, make a commit and run `git-big-picture -o our-project.png`

File: Library/Formula/git-big-picture.rb [hunk 3]
```diff
@@ -0,0 +1,18 @@
+require 'formula'
```
Comment (DomT4) [inline Library/Formula/git-big-picture.rb]: Redundant line.

File: Library/Formula/git-big-picture.rb [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+
+class GitBigPicture < Formula
+  homepage 'https://github.com/esc/git-big-picture'
```
Comment (DomT4) [inline Library/Formula/git-big-picture.rb]: Double quotes everywhere please :)

**Comment by vicamo** [comment]
@mikemcquaid , @DomT4 thank you for the comments, I'll update the pr later.

**Comment by DomT4** [comment]
If still interested please fix up any issues highlighted by `brew audit --strict` and then ping me, Thanks!

**Comment by DomT4** [comment]
Closing due to no response. Will welcome a new PR with the issues highlighted addressed. Thanks for your submission to Homebrew!

**Pull request closed without merging**